install.bash

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Unoffical Bash "strict mode"
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+#ORIGINAL_IFS=$IFS
+IFS=$'\t\n' # Stricter IFS settings
+
+asdf_install_dir=${ASDF_INSTALL_DIR:-"$HOME/.asdf"}
+
+git clone https://github.com/asdf-vm/asdf.git "${asdf_install_dir}"
+# checkout latest tag in repo in another dir
+# credit: https://stackoverflow.com/a/6073628/7911479 and https://stackoverflow.com/a/31811385/7911479
+git --git-dir "${asdf_install_dir}/.git" --work-tree "${asdf_install_dir}" checkout "$(git describe --abbrev=0 --tags)" --quiet
+
+printf "\n%s\n" "asdf setup"
+printf "%s\t\t\t%s\n" "1: install asdf" "completed!"
+printf "%s\t%s\n" "2: add asdf to your shell" "https://asdf-vm.com/#/core-manage-asdf-vm"
+printf "%s\t\t\t%s\n" "3: add a plugin" "https://asdf-vm.com/#/core-manage-plugins"
+printf "%s\t%s\n" "4: install a tool version" "https://asdf-vm.com/#/core-manage-versions"
```

Comment:
I'm not sure that providing links provides much benefit to the user. Why can't we just tell the user what do to add asdf to their shell? If this script is invoked by the package manager we should be able to tell the user exactly what do to for their specific OS/package manager combination.

Reply:
I think that solution is unmaintainable for us. The number of permutations is making the docs difficult to update as is. `nvm` does this and their script is 100s of lines of conditional shell scripting which it almost certainly doesn't work in all cases.

Reply:
>I'm not sure that providing links provides much benefit to the user

The idea was to capture users who hear about `asdf` and just run their package manager of choices installation, eg `brew install asdf`, and haven't ever come to the documentation site. These users have find that it doesn't work, they were given no further instructions, and instead of looking for a documentation site to see if there is something they missed, they check GitHub for issues where people used the same package manager and report they themselves are having issues.

I would like to show these people 2 things:
1. there's more steps before you can use `asdf`
2. these steps are on our documentation site

Reply:
Given we don't need to inform users who are installing from the documentation site, we could leave the docs as is, and only use the install script in Homebrew and co after they clone the repo as they currently do.